components/bl-parallax-component/src/index.js

```diff
@@ -9,7 +9,7 @@ export default function Parallax({ component, pods }) {
   const backdropRef = useRef();
   const containerRef = useRef();
 
-  useAnimation(backdropRef, containerRef, strength, display);
+  useAnimation(backdropRef, containerRef, Math.min(strength, 2000), display);
```

Comment:
Magic number- create constant for that. Also if 2000 is maximum possible, we should write it in docs.